### DISCUSSION TITLE
How to get result of miCLIP?

### DISCUSSION THREAD

**Discussion by thadasps** [GoekeLab/m6anet#122] [Q&A] [answered]
Hi, I try to generate the ROC curve follow with your figure 1d. but I have only result of m6A sites from m6ACE-seq (Downloaded from xPore). Can you point me or suggest the way how to get the result from miCLIP? (I can't find any result from miCLIP's paper)

Thank you

**Comment by Codezy99** [comment]
Maybe you can check out this paper: https://academic.oup.com/nar/article/49/16/e92/6307904?login=false 
And their data can be found at here: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?&acc=GSE163500

**Comment by kristinrma** [comment, marked as answer]
Hi @thadasps, we used the two m6A sites tables from the Supplementary section of the miCLIP paper here https://www.nature.com/articles/nmeth.3453#Sec30. If a site was included in either library, we considered it a modified site for training and testing purposes.
